monai/transforms/croppad/array.py

```diff
@@ -303,7 +304,7 @@ class RandSpatialCrop(Randomizable):
             if its components have non-positive values, the corresponding size of input image will be used.
         random_center: crop at random position as center or the image center.
         random_size: crop with random size or specific size ROI.
-            The actual size is sampled from `randint(roi_size, img_size)`.
+            if True, the actual size is sampled from `randint(roi_size, max_roi_size)`.
```

Comment:
the `randint` API from  numpy doesn't include the higher value, the interval is [low, high), could you help check the related edge cases..

Reply:
sure, updated in the latest commit.
Thanks.